adb_graphics/figures/maps.py

```diff
@@ -247,6 +248,7 @@ def _colorbar(self, cc, ax):
 
     @utils.timer
     def draw(self, show=False):
+    # pylint: disable=too-many-locals
```

Comment:
Would you mind either adding this to the end of line 250 or indenting it to be consistent with the code inside the method? Either way is good.